lib/matplotlib/axes/_secondary_axes.py

```diff
@@ -13,7 +14,7 @@ class SecondaryAxis(_AxesBase):
     General class to hold a Secondary_X/Yaxis.
     """
 
-    def __init__(self, parent, orientation, location, functions, **kwargs):
+    def __init__(self, parent, orientation, location, functions, transform, **kwargs):
```

Comment:
This should likely have a default to avoid breaking any prior calls which do not pass a transform

Reply:
Makes sense, done